draft-dnsop-deleg-transport.md

```diff
@@ -208,31 +217,33 @@ alpn=doq tlsa=disabled mandatory=alpn
 
-## Deployment Considerations
+## Resolution procedure
 
-The DELEG and SVCB records intends to replace the NS record while also adding additional functionality in order to support additional transports for the DNS. Below are discussions of considerations for deployment.
+As specified in {{?I-D.draft-dnsop-deleg}}, the ServiceMode DELEG records provide a weak signal from the operator about which transport mechanisms they would most prefer clients to use, but it is not mandatory that resolvers try the delegated servers in priority order.  It might not even be possible for a resolver to do, such as if the highest priority record specifies an alpn that the resolver does not implement, but a lower priority record has an alpn that it can use.
 
-### Availability
+Thus, as with authority server selection for NS records, a resolver is free to use its own policy implementation to determine which servers to continue to use for resolution.
 
-If a zone operator removes all NS records before DELEG and SVCB records are implemented by all clients, the availability of their zones will be impacted for the clients that are using non-supporting resolvers. In some cases, this may be a desired quality, but should be noted by zone owners and operators.
+If a connection error occurs when a resolver attempts to access a nameserver delegated to by a DELEG or SVCB record, such as with a certificate mismatch or by being unreachable, the resolver SHOULD attempt to connect to the other nameservers in the DELEG/SVCB RRset to until either exhausting the list or the resolver's policy indicates that they should treat the resolution as failed.
 
-# Privacy Considerations
+For resolvers that strongly favor privacy, operators may wish to return a SERVFAIL when all attempts to use privacy-enhanced alpns in the DELEG/SVCB resolution process have failed. More permissive resolvers can fall back to using the legacy DNS resolution process via NS records.
 
-All of the information handled or transmitted by this protocol is public information published in the DNS.
+## Deployment Considerations
 
-# Security Considerations
+DELEG and SVCB records are intended to be the best method for modern resolvers to use delegation information for zones.
 
-TODO: Fill this section out
+Realistically, however, legacy resolvers will continue to exist on the Internet for a long time to come.  If a zone operator removes all NS records, the availability of their zones will be impacted for the clients that are using non-supporting resolvers. In some cases, this may be a desired quality, but should be noted by zone owners and operators.
 
-## Resolution procedure
+# Privacy Considerations
 
-TODO: Define how resolver selects which server connect to, especially when faced with selection of multiple transports. I's probably resolver's policy decision, but it should be said explicitly.
+All of the information transmitted by this protocol is public information published in the DNS.
```

Comment:
I don't think queries are meant to be "published" in the DNS.

Reply:
I also thought the sentence as a little awkward, but was working with some prior text.   I agree regarding queries.   This sentence could maybe be improved with "transmitted by authoritative servers", but the section needs work anyway.

Reply:
```suggestion
Negotiation of alternative transports with DELEG allows the use of authenticated encryption between resolvers and authoritative DNS servers.  This improves privacy by concealing the QNAME and QTYPE from a network observer.  This is especially beneficial when a resolver is only shared by a small number of users.
```